### ISSUE TITLE
ClickHouse disk type with first upper letter

### ISSUE THREAD

**Issue by orloffv** [Altinity/clickhouse-operator#1411]
https://github.com/Altinity/clickhouse-operator/blob/bbbf66a8e0fbbcf36b787a63eceeaca37e0ec272/pkg/apis/metrics/clickhouse_metrics_fetcher.go#L119

<img width="1061" alt="Screenshot 2024-05-03 at 11 09 26" src="https://github.com/Altinity/clickhouse-operator/assets/504986/b103ced9-9ab5-4e52-bad4-77310dc5894e">

i added new disk for example:
```
	"storage_configuration/disks/disk1/type": "local"
	"storage_configuration/disks/disk1/path": "/var/log/clickhouse-server/"
	"storage_configuration/disks/disk1/keep_free_space_bytes": 100000
```

and type is "Local"
the metrics aren't fetched.

**Comment by orloffv** [comment]
clickhouse version is 24.4.1.2088

**Comment by orloffv** [comment]
https://fiddle.clickhouse.com/4ca0bd51-54d4-45c1-9896-0d7c2bafdc39 24.2.3 - local
https://fiddle.clickhouse.com/4ca0bd51-54d4-45c1-9896-0d7c2bafdc39 24.3.1 - Local

**Comment by Slach** [comment]
thanks for reporting

**Comment by alex-zaitsev** [comment]
Fixed in 0.24.0

**Comment by alex-zaitsev** [comment]
Fix is delivered in https://github.com/Altinity/clickhouse-operator/releases/tag/release-0.23.7 as well
